src/segment-loader.js

```diff
@@ -6,13 +6,11 @@ import {getMediaIndexForTime_ as getMediaIndexForTime, duration} from './playlis
 import videojs from 'video.js';
 import SourceUpdater from './source-updater';
 import {Decrypter} from './decrypter';
+import Config from './config';
```

Comment:
`Config` will be the same object every time so that each source will start off with the config of it's predecessor instead of the default config.